integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -59,6 +59,16 @@ spec:
       {{- end }}
     {{- end }}
     spec:
+      {{ if .Values.hostAliases }}
+      hostAliases:
+        {{- range .Values.hostAliases }}
+        - ip: {{ .ip }}
+          hostnames:
+            {{- range .hostnames }}
+            - {{ . }}
+            {{- end }}
+        {{- end }}
+      {{ end }}
```

Comment:
This part is the same code as the one in worker YAML. You can extract this shared code into https://github.com/Alluxio/alluxio/blob/master/integration/kubernetes/helm-chart/alluxio/templates/_helpers.tpl

Reply:
@jiacheliu3 OK~ I'll move these to `_helpers.tpl`